### PR TITLE
Feat radio img button

### PR DESCRIPTION
**Pull request by GranzonLin**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: examples/docs/zh-CN/radio-img-button.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```
+:::
+
+### Radio-img-button Attributes
+| 参数      | 说明    | 类型      | 可选值       | 默认值   |
+|---------- |-------- |---------- |-------------  |-------- |
+| value / v-model | 绑定值 | string / number / boolean | — | — |
+| disabled  | 是否禁用    | boolean   | — | false   |
+| name | 原生 name 属性 | string    |      —         |     —    |
+| width | 按钮宽度 | string | — | 160px |
```
Comment (Curry30XTY) [inline examples/docs/zh-CN/radio-img-button.md]: width设置要支持一下number类型

File: examples/docs/zh-CN/radio-img-button.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+### 横向布局
+
+当设置为横向布局，记住一定要设置width
+
+:::demo 默认只有两种布局，设置为横向布局时只需要去掉vertical属性或者设置其值为false即可
+```html
+<template>
+   <div style="display: flex">
+       <el-radio-img-button v-model="value" imgUrl="http://img5.imgtn.bdimg.com/it/u=3987907653,720009510&fm=26&gp=0.jpg" label="1" width="586px" text="radio-img-button" description="此处风景优美，令人身心向往；但是却带一股寒冷之气，恰恰还是那些市井之地，才充满生活气息；不知是知了还是蛐蛐的叫声，使得夜晚更加的宁静安逸"></el-radio-img-button>
```
Comment (Curry30XTY) [inline examples/docs/zh-CN/radio-img-button.md]: 图片地址换个可在例子上显示的  现在的图片显示不出来

File: examples/docs/zh-CN/radio-img-button.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+### 按钮文字与描述
+
+与radio不同的是，如果我们需要给按钮添加文字和描述，则用text和description
+
+:::demo 值得注意的是，description只有在横向布局时才有效
+```html
+<template>
+   <div style="display: flex">
+       <el-radio-img-button v-model="value" imgUrl="http://img5.imgtn.bdimg.com/it/u=3987907653,720009510&fm=26&gp=0.jpg" label="1" width="310px" text="按钮文字" description="按钮描述"></el-radio-img-button>
```
Comment (Curry30XTY) [inline examples/docs/zh-CN/radio-img-button.md]: 例子可以多放一两个radio

File: examples/docs/zh-CN/radio-img-button.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+| 参数      | 说明    | 类型      | 可选值       | 默认值   |
+|---------- |-------- |---------- |-------------  |-------- |
+| value / v-model | 绑定值 | string / number / boolean | — | — |
+| disabled  | 是否禁用    | boolean   | — | false   |
+| name | 原生 name 属性 | string    |      —         |     —    |
+| width | 按钮宽度 | string | — | 160px |
+| imgUrl | 图片地址 | string | — |  — |
+| text | 按钮文字 | string | — |  — |
+| description | 按钮描述 | string | — |  — |
+
```
Comment (Curry30XTY) [inline examples/docs/zh-CN/radio-img-button.md]: 下面src两张图片删除掉

File: test/unit/specs/radio-img-button.spec.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { createTest, destroyVM } from '../util';
+import RadioImgButton from 'packages/radio-img-button';
+
+describe('RadioImgButton', () => {
+  let vm;
+  afterEach(() => {
+    destroyVM(vm);
+  });
+
```
Comment (Barretem) [inline test/unit/specs/radio-img-button.spec.js]: 需要补测试用例

**Pull request merged**